media/livestream/create_channel_with_backup_input.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package livestream
+
+// [START livestream_create_channel_with_backup_input]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"github.com/golang/protobuf/ptypes/duration"
+
+	livestream "cloud.google.com/go/video/livestream/apiv1"
+	livestreampb "google.golang.org/genproto/googleapis/cloud/video/livestream/v1"
+)
+
+// createChannelWithBackupInput creates a channel with a failover backup input.
+func createChannelWithBackupInput(w io.Writer, projectID string, location string, channelID string, primaryInputID string, backupInputID string, outputURI string) error {
+	// projectID := "my-project-id"
+	// location := "us-central1"
+	// channelID := "my-channel"
+	// primaryInputID := "my-primary-input"
+	// backupInputID := "my-backup-input"
+	// outputURI := "gs://my-bucket/my-output-folder/"
+	ctx := context.Background()
+	client, err := livestream.NewClient(ctx)
+	if err != nil {
+		return fmt.Errorf("NewClient: %v", err)
+	}
+	defer client.Close()
+
+	req := &livestreampb.CreateChannelRequest{
+		Parent:    fmt.Sprintf("projects/%s/locations/%s", projectID, location),
+		ChannelId: channelID,
+		Channel: &livestreampb.Channel{
+			InputAttachments: []*livestreampb.InputAttachment{
```

Comment:
nit: To help with the horizontal scroll, this could be pulled out in a variable declared above.

Reply:
Done